nntrainer/include/neuralnet.h

```diff
@@ -52,6 +52,16 @@ namespace nntrainer {
  */
 typedef enum { NET_KNN, NET_REG, NET_NEU, NET_UNKNOWN } NetType;
 
+/**
+ * @brief     Statistics from running or training a model
+ */
+typedef struct _run_stats {
+  float accuracy; /** accuracy of the model */
```

Comment:
I thought we agreed not to use variable which start with "_".

Reply:
Updated to `RunStats_`.